src/Controller/Order/Cancel/Cancel.php

```diff
@@ -208,8 +218,18 @@ public function cancelItem($orderID, $itemID)
 					$payable->setPayableAmount($item->gross);
 					$payable->setTax($item->getTax());
 
+					$gateway = null;
+					foreach ($this->_order->payments as $p) {
+						$gateway = $this->get('payment.gateway.loader')->getGatewayByPayment($p->payment);
```

Comment:
The payment gateway loader does not exist in this module, you will need to move it over from ecommerce